test/functional/p2p_opportunistic_1p1c.py

```diff
@@ -376,6 +377,46 @@ def test_other_parent_in_mempool(self):
         assert parent_low["txid"] in node_mempool
         assert child["txid"] in node_mempool
 
+    @cleanup
+    def test_1p1c_on_1p1c(self):
+        self.log.info("Test that opportunistic 1p1c works when part of a 4-generation chain (1p1c chained from a 1p1c)")
+        node = self.nodes[0]
+
+        # Prep 2 generations of 1p1c packages to be relayed
+        low_fee_great_grandparent = self.create_tx_below_mempoolminfee(self.wallet)
+        high_fee_grandparent = self.wallet.create_self_transfer(utxo_to_spend=low_fee_great_grandparent["new_utxo"], fee_rate=20*FEERATE_1SAT_VB)
+
+        low_fee_parent = self.create_tx_below_mempoolminfee(self.wallet, utxo_to_spend=high_fee_grandparent["new_utxo"])
+        high_fee_child = self.wallet.create_self_transfer(utxo_to_spend=low_fee_parent["new_utxo"], fee_rate=20*FEERATE_1SAT_VB)
+
+        peer_sender = node.add_p2p_connection(P2PInterface())
+
+        # The 1p1c that spends the confirmed utxo must be received first. Afterwards, the "younger" 1p1c can be received.
+        for package in [[low_fee_great_grandparent, high_fee_grandparent], [low_fee_parent, high_fee_child]]:
+            # Aliases
+            low_fee_parent, high_fee_child = package
```

Comment:
In c8d96e23f734e275dfee840df49b1f4cc78cd4b9 " test: add chained 1p1c propagation test "

nit: it is a bit confusing to reuse the `low_fee_parent` and `high_fee_child` variables here.